website/www/site/content/en/documentation/sdks/python.md

```diff
@@ -48,6 +48,11 @@ language-specific implementation guidance.
 
 ## Using Beam Python SDK in your ML pipelines
 
+To use the Beam Python SDK with your machine learning pipelines, you can either use the RunInference API or TensorFlow.
```

Comment:
change: use the RunInference API or TensorFlow
to: use the RunInference API for PyTorch and Sklearn models. If using Tensorflow model you can make use of the library from tfx_bsl. Further integrations for tensorflow are planned.

Reply:
Updated, but I'm not sure if we'll need to update this again based on the email thread?